content/working_with/console/widgets/managers.md

```diff
@@ -1,32 +1,36 @@
 ---
 layout: bt_wiki
-title: Cloudify Managers Management
+title: Spire Manager
 category: Cloudify Console
 draft: false
 ---
 
-Displays the list of the deployments created using [Cloudify Manager of Managers plugin](https://github.com/Cloudify-PS/manager-of-managers) in the current tenant, according to the user’s permissions. The data is displayed in a table.
+Displays the list of the deployments created using [Cloudify Spire plugin](https://github.com/cloudify-cosmo/cloudify-spire-plugin) in the current tenant, according to the user’s permissions. The data is displayed in a table.
 
-![managers]( /images/ui/widgets/managers.png )
+![spire-manager]( /images/ui/widgets/spire-manager.png )
 
 ##### Presented data
 
-You can see IP addresses, names 
-and status of all managers in the cluster. 
+You can see IP addresses, names and status of the cluster created by Spire deployment. 
 
-Detailed status about specific is presented after hovering the status icon:
+Detailed status about specific cluster is presented after hovering the status icon:
+
+![managers]( /images/ui/widgets/spire-manager-status.png )
```

Comment:
This png does not make sense because if all nodes are red, shouldnt the heart icon be red as well?
Lets set the png to include the managers in yellow state and the DB+broker in green state.

Reply:
Yeah, you're right.
Fixed using mock.